spec/models/token_spec.rb

```diff
@@ -44,30 +38,26 @@
   end
 
   describe 'acknowledged?' do
-    it 'should return true if accepted' do
```

Comment:
I think we should keep these tests as well as the new ones, Think they'e testing the acknowledged? method rather than the acknowledged attribute we are testing with the new tests

Reply:
All done.